test/org/sagebionetworks/bridge/services/StudyServiceMockTest.java

```diff
@@ -722,7 +722,13 @@ public void changingResetPasswordTemplateUpdatesDirectory() {
     public void changingEmailVerificationEnabledUpdatesDirectory() {
         assertDirectoryUpdated(study -> study.setEmailVerificationEnabled(false));
     }
-    
+
+    @Test
+    public void changingDisableExportUpdatesDirectory() {
+        assertDirectoryUpdated(study -> study.setDisableExport(true));
+    }
```

Comment:
If this test succeeds, then something is very wrong, because we shouldn't be updating the Stormpath directory when we update the DisableExport flag.